### PR TITLE
Update communication links in footer

### PR DESCRIPTION
**Pull request by settermjd**
This PR deprecates the mailing list and IRC in favour of talk.owncloud.com. Requested in https://github.com/owncloud/docs/issues/1181.

### CODE REVIEW

File: src/partials/footer-content.hbs [hunk 1]
```diff
@@ -29,11 +29,10 @@
         <div class="footer-nav">
           <h4>Interact</h4>
           <ul id="menu-interact" class="menu">
-            <li class="menu-irc-channel"><a href="https://webchat.freenode.net/?channels=owncloud">IRC Channel</a></li>
-            <li class="menu-mailing-list"><a href="https://mailman.owncloud.org/mailman/listinfo/">Mailing List</a></li>
-            <li class="menu-forums"><a href="https://central.owncloud.org">Forums</a></li>
-            <li class="menu-bug-tracker"><a href="https://owncloud.org/community">Get involved</a></li>
-            <li class="menu-promote"><a href="https://owncloud.org/promote/">Discuss ownCloud with others</a></li>
+            <li class="menu-mailing-list"><a href="https://talk.owncloud.com" class="oc-footer-nav-link">owncloudTalk (Rocket.Chat)</a></li>
```
Comment (tboerger) [inline src/partials/footer-content.hbs]: I would say label it just `Rocket.Chat`

File: src/partials/footer-content.hbs [hunk 2]
```diff
@@ -29,11 +29,10 @@
         <div class="footer-nav">
           <h4>Interact</h4>
           <ul id="menu-interact" class="menu">
-            <li class="menu-irc-channel"><a href="https://webchat.freenode.net/?channels=owncloud">IRC Channel</a></li>
```
Comment (tboerger) [inline src/partials/footer-content.hbs]: What happened to the IRC channel? Is it deprecated?

Comment (settermjd) [reply src/partials/footer-content.hbs]: As far as I know, yes. Is it not?

Comment (settermjd) [reply src/partials/footer-content.hbs]: According to @lefherz and @michaelstingl the channel was the subject of intense spam and is little used nowadays.

**Pull request merged**